force-app/main/default/classes/CellOrderTriggerHelper.cls

```diff
@@ -0,0 +1,35 @@
+public without sharing class CellOrderTriggerHelper {
```

Comment:
It should be `with sharing` because we work with the items that I have access to.